README.md

```diff
@@ -119,8 +119,8 @@ Our approach for client libraries is modular. Each sub-library is a standalone i
     <td><a href="https://github.com/supabase-community/functions-csharp" target="_blank" rel="noopener noreferrer">functions-csharp</a></td>
   </tr>
   <tr>
-    <td>Dart (Flutter)</td>
-    <td><a href="https://github.com/supabase/supabase-dart" target="_blank" rel="noopener noreferrer">supabase-dart</a></td>
+    <td>Flutter</td>
+    <td><a href="https://github.com/supabase/supabase-flutter" target="_blank" rel="noopener noreferrer">supabase-flutter</a></td>
```

Comment:
Looks like there are a bunch of links that'll need to be updated with this change too. Seeing quite a few results for `git grep -i 'supabase-dart'`.

Reply:
Ah, yeah. I will replace those that are appropriate. Am I correct to say that I can ignore anything in the `apps/temp-docs`?

Reply:
Assuming I can ignore temp-docs, done!

Reply:
Yep!